Makefile

```
@@ -0,0 +1,123 @@
+
+# Variables
+PACKAGE_NAME := @maximai/maxim-js
+DIST_DIR := dist
+NODE_MODULES := node_modules
+
+# Default target
+.DEFAULT_GOAL := help
+
+# Help target
+.PHONY: help
+help:
```

checkmake (0.2.2)

[warning] Target body for "help" exceeds allowed length of 5 (14).

(maxbodylength)